include/z64scene.h

```diff
@@ -2,14 +2,10 @@
 #define Z64SCENE_H
 
 #include "z64.h"
+#include "z64dma.h" // for RomFile
```

Comment:
Seems fine as is, but perhaps just forward declare `RomFile`?

Reply:
you can't, the struct is used (embedded) inside `SceneTableEntry`
we can only forward declare structs when the compiler doesn't need to know about their layout, i.e. with pointer types